.travis.yml

```diff
@@ -2,6 +2,9 @@ language: generic
 services: docker
 
 script:
+  - >
+    docker run -v=$(pwd):/app --workdir=/app
```

Comment:
This would have fit on a single line.

Reply:
Filed https://github.com/coala/rultor-python/issues/32 :+1:

Reply:
@jayvdb @Makman2 Just didn't want to insist too much ;)

@aashutoshrathi But I told you in https://github.com/coala/rultor-python/pull/27#discussion_r148964949 so go ahead to your next PR for https://github.com/coala/rultor-python/issues/32 :)